packages/location-management/src/components/LocationUnitView/tests/table.test.tsx

```diff
@@ -0,0 +1,101 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { Router } from 'react-router';
+import { history } from '@onaio/connected-reducer-registry';
+import Table, { TableData } from '../Table';
+
+describe('Location-module/locationunit', () => {
+  const tableData: TableData[] = [];
+  for (let i = 1; i < 5; i++) {
+    tableData.push({
+      id: i.toString(),
+      key: i.toString(),
+      name: `Edrward ${i}`,
+      geographicLevel: i,
+    });
+  }
+
+  it('renders without crashing', () => {
+    const wrapper = mount(
+      <Router history={history}>
+        <Table setDetail={() => jest.fn()} accessToken="hunter 2" data={tableData} />
+      </Router>
+    );
+    expect(wrapper.props()).toMatchSnapshot();
```

Comment:
I also feel the snapshot is not required here...Also, for any other similar snapshot, let's target the element of interest, so that we don't include the `Router`, `Provider` e.t.c in the snapshot e.g `wrapper.find('Table')`